src/io/io.ts

```diff
@@ -15,27 +15,37 @@
  * =============================================================================
  */
 
-import {browserDownloads, browserFiles} from './browser_files';
+// Importing loca_storage and indexed_db is necessary for the routers to be
+// registered.
+import './indexed_db';
+import './local_storage';
+
+import {browserFiles} from './browser_files';
 import {browserHTTPRequest} from './browser_http';
-import {browserIndexedDB} from './indexed_db';
 import {decodeWeights, encodeWeights} from './io_utils';
-import {browserLocalStorage} from './local_storage';
+import {IORouterRegistry} from './router_registry';
 // tslint:disable-next-line:max-line-length
 import {IOHandler, LoadHandler, ModelArtifacts, SaveConfig, SaveHandler, SaveResult, WeightsManifestConfig, WeightsManifestEntry} from './types';
 import {loadWeights} from './weights_loader';
 
+const registerSaveRouter = IORouterRegistry.registerSaveRouter;
+const registerLoadRouter = IORouterRegistry.registerLoadRouter;
+const getSaveHandler = IORouterRegistry.getSaveHandler;
+const getLoadHandler = IORouterRegistry.getLoadHandler;
+
 export {
-  browserDownloads,
```

Comment:
I think we might still want to export these for symmetry purpose.  I think its useful to the mental model to expose the non-sugared version that matches the pattern that the non-sugared handlers use.